utils/rtl_433/Makefile

```diff
@@ -0,0 +1,45 @@
+# 
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtl_433
+PKG_VERSION:=6531ea4
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/merbanan/rtl_433.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=6531ea48a7933ac8289724672059e54fd8aad8eb
```

Comment:
Need:

```
WARNING: PKG_MIRROR_HASH is missing, set to f96301c8d7e87605b113aad4493f15b63a675fea6f00642b0606de5a3a858746
```

Reply:
Ok will do

Reply:
done

Reply:
I think `PKG_HASH` is sufficient.

Reply:
Is updated :)

Reply:
Sorry, my mistake.  For retrieving a tarball, use `PKG_HASH`.  For retrieving a repository via `git` or `svn`, you do need `PKG_SOURCE_HASH`.  Don't ask me why.

So you can drop the previous commit in a rebase and force push.

Reply:
You didn't mean PKG_MIRROR_HASH?

Reply:
Right, sorry.